shared/tracker/header.render.desktop.js

```diff
@@ -43,79 +33,53 @@ export default class HeaderRender extends Component {
       case 'followed':
       case 'refollowed':
       case 'unfollowed':
-        headerStyle = styles.headerSuccess
         headerTextStyle = styles.headerTextNormal
         headerText = this.props.reason
         break
       case 'error':
-        headerStyle = styles.headerWarning
         headerTextStyle = styles.headerTextWarning
     }
 
     return (
-      <div style={styles.outer}>
-        <div style={{...styles.header, ...headerStyle}}>
-          <div style={{...styles.header, ...headerStyle, height: 48, zIndex: 2, opacity: isWarning ? 1 : 0, backgroundColor: globalColors.yellow}}/>
-          <Text type='BodySemibold' lineClamp={2} style={{...styles.text, ...headerTextStyle, flex: 1, zIndex: isWarning ? 2 : 'inherit'}}>{headerText}</Text>
-          <Icon type='fa-close' style={styles.close}
-            onClick={() => this.props.onClose()}
-            onMouseEnter={() => this.closeMouseEnter()}
-            onMouseLeave={() => this.closeMouseLeave()} />
-        </div>
+      <div style={{...styles.header, ...headerStyle}}>
+        <Text type='BodySemibold' style={{...styles.text, ...headerTextStyle}}>{headerText}</Text>
+        <Icon type='fa-close' style={styles.close}
+          onClick={() => this.props.onClose()}
+          onMouseEnter={() => this.closeMouseEnter()}
+          onMouseLeave={() => this.closeMouseLeave()} />
       </div>
     )
   }
 
   closeMouseEnter () {
-    this.setState({showCloseWarning: true})
```

Comment:
dumb components shouldn't have state. also this state is now shared between header and bio